### PR TITLE
Minor improvements to OpenGitHubIssue

### PR DESCRIPTION
**Pull request by eguiraud**
Hi, I love the new `.gh` command (thank you very much @ferdymercury !).

Just a couple of minor improvement suggestions:
- remove extra newline from auto-filled GH issue
- log to terminal that a new browser tab should have been opened

### CODE REVIEW

**Comment by ferdymercury** [comment]
Thanks! May I also suggest formatting root-version textarea as monospace?
See https://github.com/root-project/root/issues/13062

File: core/base/src/TApplication.cxx [hunk 1]
```diff
@@ -1060,9 +1061,11 @@ void TApplication::OpenGitHubIssue(const TString &type)
     } else if (type == "feature") {
     OpenInBrowser("\"https://github.com/root-project/root/issues/new?labels=new+feature&template=feature_request.yml\"");
 } else {
-       Warning("OpenGitHubIssue", "cannot find \"%s\" as type for a GitHub issue\n"
-               "Available types are 'bug', 'feature' or 'improvement'.", type.Data());
+       Warning("OpenGitHubIssue", "Cannot find GitHub issue type \"%s\".\n"
+               "Available types are 'bug', 'feature' and 'improvement'.", type.Data());
     }
+
+   Info("OpenGitHubIssue", "A new tab should have opened in your browser.");
```
Comment (ferdymercury) [inline core/base/src/TApplication.cxx:1068]: This should not be printed if the Warning on 1064 was printed

Comment (ferdymercury) [reply core/base/src/TApplication.cxx:1068]: Maybe it could be moved to the OpenInBrowser function?

File: core/base/src/TApplication.cxx [hunk 2]
```diff
@@ -985,8 +985,9 @@ TString TApplication::GetSetup()
    TString setup = "";
    for (auto& line : lines) {
       setup.Append(line);
-      setup.Append("\n");
+      setup.Append('\n');
    }
+   setup.Chop(); // trim final `\n`
```
Comment (ferdymercury) [inline core/base/src/TApplication.cxx:990]: On the long run, it's maybe better to define a join function? Like https://stackoverflow.com/a/71983949

File: core/base/src/TApplication.cxx (hunk 1 above)
Comment (eguiraud) [reply core/base/src/TApplication.cxx:1068]: of course, thank you, moved it up (I prefer keeping the logging in the outer layer)

File: core/base/src/TApplication.cxx (hunk 2 above)
Comment (eguiraud) [reply core/base/src/TApplication.cxx:990]: that's a feature request for TString I guess

File: core/base/src/TApplication.cxx [hunk 3]
```diff
@@ -1059,9 +1060,10 @@ void TApplication::OpenGitHubIssue(const TString &type)
       OpenInBrowser("\"https://github.com/root-project/root/issues/new?labels=improvement&template=improvement_report.yml&root-version="+FormatHttpUrl(GetSetup())+"\"");
     } else if (type == "feature") {
     OpenInBrowser("\"https://github.com/root-project/root/issues/new?labels=new+feature&template=feature_request.yml\"");
+    Info("OpenGitHubIssue", "A new tab should have opened in your browser.");
```
Comment (ferdymercury) [inline core/base/src/TApplication.cxx:1063]: Why only here and not when bug or improvement?

Comment (eguiraud) [reply core/base/src/TApplication.cxx:1063]: because indentation is weird in this function and I can't read code...

Comment (eguiraud) [reply core/base/src/TApplication.cxx]: how about now? :sweat_smile: sorry for all the noise

**Review by ferdymercury — APPROVED** [review]

**Comment by ferdymercury** [comment]
Thanks!! LGTM

**Pull request merged**